pepper-apis/studybuilder-cli/studies/singular/patches/ddp-8554-dependent-validations.conf

```diff
@@ -0,0 +1,41 @@
+{
+  "validations": [
+    # When using the Enroll My Adult Dependent flow, if a user enters an age at or above the AOM for their region in
```

Comment:
I don't think these comments reflect what we are checking in the validation. 
Updated validations have no reference to prequal.

Reply:
This is copy paste from the Jira ticket description that corresponds to the expression we figured out.

Reply:
Anyway, I changed format criteria